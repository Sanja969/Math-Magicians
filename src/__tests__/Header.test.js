import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';
import Header from '../components/Header';

describe('test the Quote components', () => {
  it('matches the snapshot', () => {
    const tree = renderer
      .create(
        <Router>
          <Header />
        </Router>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
