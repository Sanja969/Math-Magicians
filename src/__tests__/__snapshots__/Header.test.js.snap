// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test the Quote components matches the snapshot 1`] = `
<nav
  className="container-fluid navbar navbar-expand-md navbar-dark bg-dark fixed-top p-2"
>
  <div
    className="container-fluid"
  >
    <a
      className="navbar-brand"
      href="https://sanja969.github.io/Math-Magicians"
    >
      <img
        alt=""
        className="d-inline-block align-text-top mx-3"
        height="24"
        src="https://cdn.pixabay.com/photo/2017/01/31/18/16/blackboard-2026160_960_720.png"
        width="30"
      />
      <span>
        Math-Magicians
      </span>
    </a>
    <button
      aria-controls="navbarTogglerDemo01"
      aria-expanded="false"
      aria-label="Toggle navigation"
      className="navbar-toggler"
      data-bs-target="#navbarTogglerDemo01"
      data-bs-toggle="collapse"
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <div
      className="collapse navbar-collapse text-end justify-content-end text-center"
      id="navbarTogglerDemo01"
    >
      <ul
        className="navbar-nav ml-auto"
      >
        <li
          data-testid={1}
        >
          <a
            aria-current="page"
            className="nav-item text-decoration-none active"
            href="/"
            onClick={[Function]}
          >
            <p
              className="nav-link pt-4"
            >
              Home
            </p>
          </a>
        </li>
        <li
          data-testid={2}
        >
          <a
            className="nav-item text-decoration-none"
            href="/calculator"
            onClick={[Function]}
          >
            <p
              className="nav-link pt-4"
            >
              Calculator
            </p>
          </a>
        </li>
        <li
          data-testid={3}
        >
          <a
            className="nav-item text-decoration-none"
            href="/quotes"
            onClick={[Function]}
          >
            <p
              className="nav-link pt-4"
            >
              Quotes
            </p>
          </a>
        </li>
      </ul>
    </div>
  </div>
</nav>
`;
